doc/plugins/grpc-transcoding-cn.md

```diff
@@ -107,3 +107,72 @@ Proxy-Connection: keep-alive
 
 这表示已成功代理。
 
+
+#### 使用 grpc-transcode 插件的 option 选项
+
+在指定 route 中，代理 grpc 服务接口:
+
+**选项清单**
+ * 枚举类型
+    > enum_as_name
+    > enum_as_value
+
+ * 64位整型
+    > int64_as_number
+    > int64_as_string
+    > int64_as_hexstring
+
+ * 使用默认值
+    > auto_default_values
+    > no_default_values
+    > use_default_values
+    > use_default_metatable
+
+  * Hooks开关
+    > enable_hooks
+    > disable_hooks
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/23 -X PUT -d '
+{
+    "methods": ["GET"],
+    "uri": "/zeebe/WorkflowInstanceCreate",
+    "service_protocol": "grpc",
+    "plugins": {
+        "grpc-transcode": {
+         "proto_id": "1",
+         "service": "gateway_protocol.Gateway",
+         "method": "CreateWorkflowInstance",
+         "option":["int64_as_string"]
```

Comment:
Missing schema to check it.

Reply:
`pb_options` is a better name.

Reply:
Schema check increased, variable name replaced with "pb_option", Test case is trying.